src/authorization/resolver/group.resolver.ts

```diff
@@ -29,6 +30,13 @@ export class GroupResolver {
     return this.groupService.getGroupById(id);
   }
 
+  @ResolveField('users')
+  async getUsersInGroupResolveField(group: Group) {
```

Comment:
It is understood that this is a Resolve Field of `Users` in the Parent `Group`

`getUsersByGroup` would suffice right?

Reply:
Yes